utam-core/src/main/java/utam/core/driver/Navigation.java

```diff
@@ -7,6 +7,9 @@
  */
 package utam.core.driver;
 
+import utam.core.framework.base.PageObject;
```

Comment:
When changing signatures of the methods, this import should become unused (see below comments).